src/OpenTelemetry/Metrics/Base2ExponentialBucketHistogram.cs

```diff
@@ -135,8 +135,8 @@ internal int Scale
 
     internal CircularBufferBuckets NegativeBuckets { get; }
 
-    internal ExponentialBucketSnapshot ExponentialBucketSnapshot =>
-        new ExponentialBucketSnapshot(this.SnapshotScale, this.SnapshotZeroCount, this.SnapshotPositiveBuckets, this.SnapshotNegativeBuckets);
+    internal ExponentialHistogramData SnapshotExponentialHistogramBuckets =>
```

Comment:
I think it's better to make this a proper method instead of a property. It would look more readable/intuitive at the call site.